app/full_text_search.py

```diff
@@ -214,10 +230,34 @@ def create_romanized_name_fields(romanize_method, **kwargs):
             fields.extend(create_fields_for_rank(field, romanized_name))
             romanized_names_list.append(romanized_name)
 
-    full_name_fields, romanized_full_names = create_full_name_without_space_fields(
-        romanize_method, kwargs['given_name'], kwargs['family_name'])
-    fields.extend(full_name_fields)
-    romanized_names_list.extend(romanized_full_names)
+    splited_full_names = []
+    full_name = kwargs['full_name']
+    if full_name and not (' ' in full_name):
+        splited_full_names = split_full_name_without_space(
```

Comment:
We want to apply this for search queries too. So how about implementing this in script_variant.romanize_japanese_name_by_name_dict instead?